client/components/domains/domain-suggestion-flag/index.jsx

```diff
@@ -15,7 +15,7 @@ const DomainSuggestionFlag = React.createClass( {
 	},
 
 	render() {
-		let newTLDs = ['.live'];
+		let newTLDs = [];
```

Comment:
`const` is a good fit here - so that the reader is sure that this is the intended value and nothing is supposed to be added after this declaration.